CONTRIBUTING.md

```diff
@@ -4,9 +4,13 @@ Please note that this project is released with a [Contributor Code of Conduct](C
 
 ## Branch Compatibility
 
+Only Akeneo long term support (LTS) versions are supported:
+
+
 | Branch | Akeneo Compatibility |
 | ------------- |:-------------:|
-| `master` | `>= 3.0.0` |
+| `master` | `>= 3.2.0` |
```

Comment:
Should be:
```md
| Branch   |  Akeneo Compatibility |
|----------|:---------------------:|
| `master` | `>= 3.2.0`            |
| `3.0`    | `>= 3.0.21 & < 3.1.0` |
| `2.X`    | `>= 2.0.5 & < 3.0.0`  |
| `2.0`    | `>= 2.0.0 & < 2.0.5`  |
| `1.X`    | `>= 1.6.0 & < 2.0.0`  |
```

Reply:
That's exactly what is done. Or am I missing something?

Reply:
For better readability the columns in the raw format should be clearly distinguishable

Reply:
Done